ios/REAUtils.h

```diff
@@ -5,3 +5,10 @@
     if (value == nil)                         \
       RCTLogError(errorMsg);                  \
   })
+
+@interface REAUtils : NSObject
```

Comment:
is there a need for this to be encapsulated within a objc class? Could just be a function, no?